tools/walletextension/frontend/src/components/modules/home/connected.tsx

```diff
@@ -25,10 +25,10 @@ const Connected = () => {
         <h3 className="text-sm text-muted-foreground my-4">
           Manage the accounts you have connected to the Ten Gateway. You can
           revoke access to your accounts at any time and request new tokens from
-          the Ten Discord.
+          the Ten Faucet.
```

Comment:
@0xjba should this be `Ten Gas Station` or `Ten Faucet`?

Reply:
So we decided to go with TEN Gas Station